LogAndStream/python_scripts/Shimmer_common/shimmer_comms_bluetooth.py

```diff
@@ -40,13 +44,20 @@ class BtCmds:
     GET_GSR_RANGE_COMMAND = 0x23
     # DEPRECATED_GET_DEVICE_VERSION_COMMAND         = 0x24
     DEVICE_VERSION_RESPONSE = 0x25  # maintain compatibility with Shimmer2/2r BtStream
+    SET_EMG_CALIBRATION_COMMAND = 0x26
+    EMG_CALIBRATION_RESPONSE = 0x27
+    GET_EMG_CALIBRATION_COMMAND = 0x28
+    SET_ECG_CALIBRATION_COMMAND = 0x29
+    ECG_CALIBRATION_RESPONSE = 0x2A
+    GET_ECG_CALIBRATION_COMMAND = 0x2B
     GET_ALL_CALIBRATION_COMMAND = 0x2C
     ALL_CALIBRATION_RESPONSE = 0x2D
     GET_FW_VERSION_COMMAND = 0x2E  # maintain compatibility with Shimmer2/2r BtStream
     FW_VERSION_RESPONSE = 0x2F  # maintain compatibility with Shimmer2/2r BtStream
     SET_CHARGE_STATUS_LED_COMMAND = 0x30
     CHARGE_STATUS_LED_RESPONSE = 0x31
     GET_CHARGE_STATUS_LED_COMMAND = 0x32
+    SET_BUFFER_SIZE_COMMAND = 0x34
```

Comment:
not supported in FW
![image](https://github.com/ShimmerResearch/shimmer3/assets/6526073/630b61eb-457a-44ce-8975-90d721ab25cd)